PixelHunter1995/InventoryLib/Inventory.cs

```diff
@@ -0,0 +1,118 @@
+﻿
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using PixelHunter1995.TilesetLib;
+
+namespace PixelHunter1995.InventoryLib
+{
+    class Inventory
+    {
+        private static readonly int X_POS = 120;
+        private static readonly int Y_POS = 160;
+        private static readonly int X_PADDING = 5;
+        private static readonly int Y_PADDING = 5;
+        private static readonly int X_ITEM_SIZE = 32;
+        private static readonly int Y_ITEM_SIZE = 32;
+        private static readonly int COLUMNS = 6;
+        private static readonly int ROWS = 2;
+
+        private List<InventoryItem> ItemList = new List<InventoryItem>();
+        private Texture2D InventoryBackground;
+        private Tileset InventoryTileset;
+
+        public void Initialize(string tilesetDirPath)
```

Comment:
Seems like the tilesetDirPath should be constant.

Also I'm not sure we need an actual "Initialize" function when we can just put that code in the constructor.